docs/docs/30-administration/10-server-config.md

```diff
@@ -17,8 +17,36 @@ services:
     environment:
       - [...]
 +     - WOODPECKER_OPEN=true
-+     - WOODPECKER_ORGS=dolores,dogpatch
+```
+
+When registration is closed, an administrator must add new users manually (with the CLI: `woodpecker-cli user add`) or by allowing specific users via `WOODPECKER_ADMIN` or `WOODPECKER_ORGS`.
+
+To close registration, but allow specific users:
+```diff
+# docker-compose.yml
+version: '3'
 
+services:
+  woodpecker-server:
+    [...]
+    environment:
+      - [...]
++     - WOODPECKER_OPEN=false
++     - WOODPECKER_ADMIN=johnsmith,janedoe
```

Comment:
As the admin option is not related to open I would explain it without mentioning the open option

Reply:
WOODPECKER_ADMIN is setting users to be admin not if they are allowed to login or not

Reply:
https://github.com/woodpecker-ci/woodpecker/blob/a958049485d0284f48ed80e4650330ebe9b0aa65/server/api/login.go#L76

> WOODPECKER_ADMIN is setting users to be admin not if they are allowed to login or not

well yes but it's also used for that - to not have the chiken-egg problem